include/seqan/index/index_bifm.h

```diff
@@ -56,6 +56,8 @@ SEQAN_CONCEPT_IMPL((Index<TText, BidirectionalIndex<FMIndex<TSpec, TConfig> > >
 template <typename TText, typename TSpec, typename TConfig>
 inline bool indexCreate(Index<TText, BidirectionalIndex<FMIndex<TSpec, TConfig> > > & index)
 {
+    indexText(index.rev) = indexText(index.fwd);
+    reverse(indexText(index.rev));
```

Comment:
If the index was created with the strings as argument, it will be reversed twice, see above.